lib/makeSequence.m

```diff
@@ -232,7 +232,7 @@
 %            seq.patternID{cPat} = currPatternID; 
             seq.patternID{cPat}     = currPatternID; 
             seq.segmCateg{cPat}     = currCategLabel; 
-            seq.onset(cPat)     = currTimePoint; 
+            seq.patternOnset(cPat)  = currTimePoint; 
             seq.pattern{1,cPat}     = currPattern; 
             seq.F0(cPat)            = currF0;
             seq.gridIOI(cPat)       = currGridIOI;
```

Comment:
would it make sense to put BIDS formatting the struct of currSeq in makeSequence.m (basically the structure would change from line to row or something like that)

Reply:
definitely makes sense.